<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWindow</class>
 <widget class="QWidget" name="LogInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1151</width>
     <height>601</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="2,3,2" columnstretch="2,4,2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="total_layout" stretch="5,1,1">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="user" stretch="2,7,3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,4,7,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name_lb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="password_lb">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>密码</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="name_le"/>
          </item>
          <item>
           <widget class="QLineEdit" name="pwd_le">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="register_btn">
            <property name="text">
             <string>新用户注册</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forgetpwd_btn">
            <property name="text">
             <string>忘记密码？</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="settings" stretch="2,3,3,2">
        <item>
         <widget class="QLabel" name="empty_left_lb">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autologin_cb">
          <property name="text">
           <string>自动登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="remempwd_btn">
          <property name="text">
           <string>记住密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="empty_right">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttons" stretch="3,5,5,4">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="empty_left_lb_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="login_btn">
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_btn">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="empty_right_lb">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>LogInWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
